robottelo.properties.sample

```diff
@@ -26,6 +26,8 @@ project=sat
 locale=en_US.UTF-8
 remote=0
 smoke=0
+# Update upstream=0 for downstream run
+upstream=1
```

Comment:
The scope of this effort is still being decided.  We will know more in few days.